packages/xod-arduino/platform/program.tpl.cpp

```diff
@@ -72,4 +72,8 @@ namespace xod {
         &storage_{{ id }}{{#unless @last }},{{/unless }}
       {{/each}}
     };
+
+    const NodeId g_defer_node_ids[DEFER_NODE_COUNT] PROGMEM = {
```

Comment:
Please, use `camelCase`. `g_` is from Hungarian notation and stands for “Global”:

```cpp
const NodeId g_deferNodeIds[DEFER_NODE_COUNT] PROGMEM = {
```